packages/hash/backend-utils/src/search/doc-types.ts

```diff
@@ -12,4 +12,9 @@ export type EntitiesDocument = {
   updatedAt: string;
   updatedByAccountId: string;
   fullTextSearch?: string;
+  belongsToPage?: {
+    entityId: string;
+    entityVersionId: string;
```

Comment:
This is to comply with the GQL `EntityRef` type, makes conversion from the search index to a `PageSearchResult` trivial

Reply:
Because as part of this PR we are only indexing the latest version of entities, perhaps instead of `EntityRef` we should be using `LatestEntityRef` in the GQL definitions for search-related things? This should let us remove `entityVersionId` here without sacrificing simplicity in that conversion process.

Reply:
Hmm, I was not aware of `LatestEntityRef`. 
This would also touch on the existing `searchPages` resolver as the search result is a shared type. 
Would this influence anything on the frontend search functionality?

Reply:
I did not implement FE search, but my guess would be that this shouldn't impact the current implementation of search (as it shouldn't really do anything with the `entityVersionId`s as far as I'm aware). I think this is worth looking into, as I'd like us to avoid passing around `entityVersionId` in places where it has no meaning.

Reply:
Understood, in that case lets create a follow-up task 👍

Reply:
Captures in [this task](https://app.asana.com/0/1200211978612931/1201892371894926/f) (_internal_)

Reply:
Thank you :)

Reply:
To note that the frontend isn't using search at the moment - it's been broken since the links refactor. 

Agree this is good to do in follow up, but when doing follow ups I wouldn't worry about breaking the frontend search.